crow/sequins2.md

```diff
@@ -5,15 +5,17 @@ nav_exclude: true
 
 # sequins v2
 
-the sequins library is designed for building sequencers, arpeggiators and all sorts of other pattern systems. this new version adds to the original with `transformers` for simple pattern synthesis, and also streamlines some syntax for character-patterns.
+The sequins library is designed for building sequencers, arpeggiators and all sorts of other pattern systems. This new version adds to the original with `transformers` for simple pattern synthesis, and also streamlines some syntax for character-patterns.
 
-## a quick refresher
+## A quick refresher
```

Comment:
@trentgill , is this a TODO or can we safely nix this header?

Reply:
this was a TODO, but i'm happy to drop it. might make sense to write-up a similar document for the original sequins features (but do i remember you already wrote something like this?)